### DISCUSSION TITLE
[Feature Request] Provide a way to filter/search for game modes.

### DISCUSSION THREAD

**Discussion by sahildave** [ChristopherKlay/StadiaEnhanced#222] [Ideas & Requests]
It is currently impossible to find a game with specific game mode, say Local co-op, unless you go to each game, scroll & see.
This can be a search feature or a filter.

This data is available in [Stadia Database](https://linktr.ee/StadiaDatabase)

**Comment by OriginaIPenguin** [comment]
@ChristopherKlay I've added a column to your database in case you decide to implement this 
![image](https://user-images.githubusercontent.com/78177972/153880060-55621bdb-2af1-46f4-aa50-8156dcd85fc2.png)

**Comment by ChristopherKlay** [comment]
@OriginaIPenguin That's pretty much perfect.

I'm currently working on `1.20.0` which is mainly focused on refactoring the code to be easier to maintain/extend functionality and performance (after the recent poll on twitter), but - unless there's any bigger complication - will likely squeeze this in as well.

I thought about adding a co-op icon to the library first, but given the lack of gaming specific icons from the material collection, i'll likely just clone a version of the existing dropdown menu and fill it with all available options to filter (:
